### DISCUSSION TITLE
Tasmota - electric meter value - switching wifi smart socket

### DISCUSSION THREAD

**Discussion by Bayern55** [arendst/Tasmota#18434] [Q&A]
Is it possible to have a socket switched only with Tasmota meter value?

e.g.
An electricity meter shows the solar power to be delivered free of charge to the supplier. Can you use it to switch a socket?

Example:

Counter –100 watts and more negatively switch on socket
Counter -99 watts and less negative → socket off
Check every 5 minutes

I am looking for a simple solution.

**Comment by barbudor** [comment]
If your meter is already integrated with Tasmota, it's just a matter of creating a rule or a script function.

**Comment by Bayern55** [comment]
It is a digital electricity meter. I read it with a Hichi reading head, which has integrated Tasmota and transmits the current power consumption with WLAN to the Fritz.box.

Which function of Tasmota is suitable for switching purposes of a socket depending on the displayed power?

**Reply by barbudor** [reply]
I already pointed to Rules or Scripting-Language
If your meter integration is using Scripting-Language, then use Scripting-Language
If done differently, Rules probably

Considering that you are giving no clues, I can't guess what would be the best approach
